drv.go

```diff
@@ -927,6 +1109,17 @@ func (d *drv) OpenConnector(name string) (driver.Connector, error) {
 // The returned connection is only used by one goroutine at a
 // time.
 func (c connector) Connect(context.Context) (driver.Conn, error) {
+    if c.PoolParams != nil {
+        pool, err := c.drv.openPool(c.PoolParams)
+        if err != nil {
+            return nil, err
+        }
+        connection := conn{drv: c.drv, timeZone: time.Local}
```

Comment:
Do we really know the server time zone here?

Reply:
I was simply following the code found in openConn() as seen [here](https://github.com/godror/godror/blob/master/drv.go#L190). Do you have a suggestion on what should be done instead?

Reply:
Ok, then leave it as is.